recipes/pytest-pylint/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pytest-pylint" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "41d6f223ae5e0a0fbb0056e826ecdb8046be2a828eba55c0d4f66cbfd7d27168" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - six
+    - pylint
+    - pytest
+    - astroid
+    - isort
+    - mccabe
+    - colorama
+    - py
+    - lazy-object-proxy
+    - wrapt
+  run:
+    - python
+    - six
+    - pylint
+    - pytest
+    - astroid
+    - isort
+    - mccabe
+    - colorama
+    - py
+    - lazy-object-proxy
+    - wrapt
+
+test:
+  imports:
+    - pytest_pylint
+
+about:
+  home: https://github.com/carsongee/pytest-pylint
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'pytest plugin for running pylint against your codebase'
+
+  description: |
+    Run pylint with pytest and have configurable rule types (i.e. Convention, Warn, and Error) fail the build. You can also specify a pylintrc file.
```

Comment:
Please keep the line below 80 characters. (Going to the newline in this block won't make an actual newline in the description.

Reply:
Done